src/fracas/syntax.ts

```diff
@@ -123,9 +131,16 @@ function _anyMaskOrEnumRx(): string {
 }
 
 function _anyIdentifierRx(fieldName: string, searchKind = SearchKind.wholeMatch): string {
-    return searchKind === SearchKind.wholeMatch ?
-        `(${_escapeForRegEx(fieldName)})(?!${RX_CHAR_IDENTIFIER})` :
-        `(${_escapeForRegEx(fieldName)}${RX_CHAR_IDENTIFIER}*)`;
+    if (searchKind === SearchKind.wholeMatch) {
+        return `(?<!${RX_CHAR_IDENTIFIER})(${_escapeForRegEx(fieldName)})(?!${RX_CHAR_IDENTIFIER})`;
+    } else {
+        // if the field name isn't empty, find anything starting with the name. Otherwise, find 1 or more identifier chars.
+        const fieldExpr = fieldName ? `(${_escapeForRegEx(fieldName)}${RX_CHAR_IDENTIFIER}*)` : `(${RX_CHAR_IDENTIFIER}+)`;
+        // // This ensures that identifiers appear either at the very beginning of an s-expression, e.g. "(some-identifier...)"
```

Comment:
This commented code is leftover from when I was trying to devise a regex that can resolve both enums declared as a list of naked identifiers, and enums declared as a list of s-expressions containing the identifier and some metadata fields like `#:tool-tip`. I never found a solution that works for both, but I haven't deleted the comment because I'd like to eventually remove the special case code for finding enum members. (See `findMembers(...)` below.)